src/app/services/views/ServiceDetailView.vue

```diff
@@ -39,9 +45,13 @@ import type { ExternalServiceSource } from '../sources'
 import AppView from '@/app/application/components/app-view/AppView.vue'
 import DataSource from '@/app/application/components/data-source/DataSource.vue'
 import RouteView from '@/app/application/components/route-view/RouteView.vue'
+import EmptyBlock from '@/app/common/EmptyBlock.vue'
 import ErrorBlock from '@/app/common/ErrorBlock.vue'
 import LoadingBlock from '@/app/common/LoadingBlock.vue'
 import type { ServiceInsight } from '@/types/index.d'
+import { useI18n } from '@/utilities'
+
+const { t } = useI18n()
```

Comment:
I don’t want to use `RouteView` as an "import all our utilities" component. I prefer just using an import.

Reply:
Using the `RouteView#t` means that the `useI18n` `TOKEN` shouldn't be "half refreshed" when you get the development autorefresh leading to weird errors, so the more we use `RouteView#t` the better. If we don't need to use `t` in a script tag I would definitely prefer to use `RouteView#t`. Do you have any reason rather than preference not to use it?

Reply:
> Using the RouteView#t means that the useI18n TOKEN shouldn't be "half refreshed" when you get the development autorefresh leading to weird errors,

What does that mean "half refreshed"? I’ve not seen any issues like that and if they exist that’s a bug in how we handle service initialization, no?

Reply:
vite knows about `imports` so while I'm not totally sure about how the refresh works, I'd noticed that when you change certain files vite's auto refresh reloads the file you changed plus any `imports` it references (`useI18n` > `utilities` > `tokens.ts`). Then you sometimes get a weird error saying it can't find the token, and you need to do a hard refresh. This is what I mean by a "half refresh" of the TOKEN.

I figured this is probably changing the symbols used in brandi for service identification, but as main.ts isn't refreshed as you didn't change it, main.ts has a reference to the old tokens. I've moved `tokens.ts` to be separate and now that RouteView is a global component and we access `t` from there I've noticed I get this weird error far less. And while I'm not totally sure yet, from trying to understand what the issue was and taking some steps to avoid the issue, I've not noticed this weird error nearly as much lately.

Not this "bug" only happens due to the partial auto refresh when changing a file during development, so not a user facing "bug".

Reply:
(I think generally though, `t` is exported from RouteView because you always need it in the Route and it saves you having to import it all the time, its always there - eventually I'd like to restrict even further were we import/use `t` as mostly `t` derived strings should be passed down form the RouteView)

Reply:
I don’t think I’ve seen this issue.

Seeing as we’ll always have i18n uses that can’t be sourced from RouteView I don’t see how avoiding i18n as an imported utility actually helps addressing the underlying issue.

Not a fan of hiding away that issue by making it less common.

Reply:
We said a while back we should mostly provide `t` strings from The Outside (i.e. outside of components), if its easier to use slots than to import `t` into a component, its more likely people will use slots.

(just so its said there are times when using `t` inside components is totally the way to go, but its less common)

Let me resolve this, I don't want this to be an issue.